### ISSUE TITLE
Add public-l to events email

### ISSUE THREAD

**Issue by lalicari** [acm-uiuc/liquid#303]

**Comment by bjryan2** [comment]
doesn't the events email go out to members-l? Isn't there a separate email for public-l?

**Comment by lalicari** [comment]
Yes. We wanted to change it so that students interested in ACM who aren't members are aware of the events.

**Comment by bjryan2** [comment]
but don't we send out the events email to members-l? I'm a little confused..

**Comment by colegleason** [comment]
We do. We want to send it to members and non members.
